IdentityCore/src/requests/broker/MSIDSSOExtensionSilentTokenRequest.m

```diff
@@ -114,9 +114,9 @@ - (void)executeRequestWithCompletion:(MSIDRequestCompletionBlock)completionBlock
             return;
         }
         
-        ASAuthorizationSingleSignOnRequest *ssoRequest = [operationRequest ssoRequestWithProvider:self.ssoProvider
-                                                                                          context:self.requestParameters
-                                                                                            error:&localError];
+        ASAuthorizationSingleSignOnRequest *ssoRequest = [self.ssoProvider createRequest];
+        [operationRequest.class operation];
```

Comment:
should this be assigned to ssoRequest.requestedOperation?

Reply:
fixed :)